tools/scripts/docs-website/create-docs-pr.js

```diff
@@ -182,8 +206,8 @@ async function readReleaseNoteFile (file) {
  * @param {boolean} dryRun skip branch creation
  */
 async function createBranch (filePath, remote, version, dryRun, email, name) {
-  fs.rmSync(filePath, { recursive: true, force: true })
-  fs.mkdirSync(filePath)
+  fse.rmSync(filePath, { recursive: true, force: true })
+  fse.mkdirSync(filePath)
```

Comment:
You can use `emptyDirSync` instead of these two commands. 
https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir-sync.md

```suggestion
  fse.emptyDirSync(filePath)
```